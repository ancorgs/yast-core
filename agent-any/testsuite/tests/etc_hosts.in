#
# hosts         This file describes a number of hostname-to-address
#               mappings for the TCP/IP subsystem.  It is mostly
#               used at boot time, when no name servers are running.
#               On small systems, this file can be used instead of a
#               "named" name server.
# Syntax:
#
# IP-Address  Full-Qualified-Hostname  Short-Hostname
#

10.10.0.48     	feynman.suse.de 	feynman
127.0.0.1	localhost localhost.suse.de

