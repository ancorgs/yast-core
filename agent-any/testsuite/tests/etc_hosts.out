([$["hostname":["feynman.suse.de", "feynman"], "ip4":168427568], $["hostname":["localhost", "localhost.suse.de"], "ip4":2130706433]])
